src/ccl_cls.c

```diff
@@ -137,22 +137,91 @@ static double cl_integrand(double lk, void *params) {
   return k*pk*d1*d2;
 }
 
-void ccl_angular_cls_limber(ccl_cosmology *cosmo,
-                           ccl_cl_tracer_collection_t *trc1,
-                           ccl_cl_tracer_collection_t *trc2,
-                           ccl_f2d_t *psp,
-                           int nl_out, double *l_out, double *cl_out,
-                           int *status) {
-  int local_status;
-  int clastatus, lind;
-  integ_cl_par ipar;
-  gsl_function F;
-  double lkmin, lkmax, l;
-  double result, eresult;
+static void integ_cls_limber_spline(ccl_cosmology *cosmo,
+				    integ_cl_par *ipar,
+				    double lkmin, double lkmax,
+				    double *result, int *status)
+{
```

Comment:
put this bracket at the end of the previous line

Reply:
This is not the style I normally use in C but I see it's been made standard in CCL now, so that's fine fine. Done!